src/Avalonia.Controls.ColorPicker/ColorSlider/ColorSlider.Properties.cs

```diff
@@ -109,14 +109,39 @@ public HsvColor HsvColor
         }
 
         /// <summary>
-        /// Gets or sets a value indicating whether the alpha component is always forced to maximum for components
-        /// other than <see cref="ColorComponent"/>.
-        /// This ensures that the background is always visible and never transparent regardless of the actual color.
+        /// Gets or sets a value indicating whether the alpha component is visible and rendered.
+        /// When false, this ensures that the gradient is always visible and never transparent regardless of
+        /// the actual color. This property is ignored when the alpha component itself is being displayed.
+        /// </summary>
+        /// <remarks>
+        /// Setting to false means the alpha component is always forced to maximum for components other than
+        /// <see cref="ColorComponent"/> during rendering. This doesn't change the value of the alpha component
+        /// in the color – it is only for display.
+        /// </remarks>
+        public bool IsAlphaVisible
+        {
+            get => GetValue(IsAlphaVisibleProperty);
+            set => SetValue(IsAlphaVisibleProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the slider adapts rendering to improve user-perception
+        /// over exactness. When true in the HSVA color model, this ensures that the gradient is always visible and
+        /// never washed out regardless of the actual color. When true in the RGBA color model, this ensures
+        /// the components always appear as red, green or blue.
         /// </summary>
-        public bool IsAlphaMaxForced
+        /// <remarks>
+        /// For example, with Hue in the HSVA color model, the Saturation and Value components are always forced
+        /// to maximum values during rendering. In the RGBA color model, all components other than
+        /// <see cref="ColorComponent"/> are forced to minimum values during rendering.
+        /// <br/><br/>
+        /// Also note this property will only adjust components other than <see cref="ColorComponent"/> during rendering.
+        /// This doesn't change the values of any components in the color – it is only for display.
+        /// </remarks>
+        public bool IsPerceptive
```

Comment:
The term "perceptive" is used here as it indicates the control should improve "user-perception". "perceptive" perhaps doesn't make the most sense in the context of the control itself though. The ColorSlider isn't "perceptive".

If anyone has concerns with this property naming please let me know and if you have a better idea. I can't really find examples of this concept anywhere.